### ISSUE TITLE
New Greek υγρός – a loanword?

### ISSUE THREAD

**Issue by nitsnatsnok** [lingdb/CoBL#206]
[New Greek υγρός ‘wet’](http://cobl.info/lexeme/7664/) is not inherited from [Ancient Greek ὑγρός ‘id.’](http://cobl.info/lexeme/18991/), but was rather “[r]e-introduced from Ancient Greek in literary registers”. What's your opinion about that, should we treat it like an ordinary loanword or as inherited (currently there are just these two ways)?

**Comment by m-scarborough** [comment]
Ok.  That's something I wrote awhile back in the early stages of doing the Modern Greek cognacy, and flagged as an issue in the November workshop, but I'm not sure if we found any satisfactory solution to encode these.

The particular issue that flags this lexeme as unusual is the fact that Modern Greek has regularly lost all initial unstressed syllables, and so where these are retained, it is evidence for re-introduction from the καθαρεύουσα 'purifying' literary register, which is imitative of Ancient Greek lexis.  My justification for encoding υγρός as a loanword is that it as a borrowing from a diglossic register, it seems to me to be, technically, a language internal loanword.

But this is not a straightforward issue.  There are two phonological variants for 'yesterday' in Modern Greek: χθες [xθe̞s] and χτες [xte̞s].  I believe I am correct in understanding that the later is the regular inherited reflex because the older Medieval Greek cluster /xθ/ regularly became /xt/, and the former has had its phonology restored/hypercorrected to [xθ] on the basis of the Ancient/καθαρεύουσα spelling.  Does that make the χθες variant a loanword?  I think probably not, because inherited χτες is the Swadesh term, so the hypercorrected form seems to be a variant.  But then, by that logic should υγρός then also be considered a similar sort of phonological hypercorrection and not be considered a register loanword?  

I am not sure of a way to be completely consistent on this issue.

**Comment by CasFre** [comment]
As @m-scarborough points out in mentioning the example of χθες, the borders between loans and inheritances seem to be not clear-cut as far as Modern Greek is concerned.
Generally, however, I would say that from a linguistic perspective cases like υγρός should rather be treated as loans than as regularly inherited lexemes, because they do not share the set of regular changes that they would have undergone if inherited. So I would take sides with @m-scarborough's initial decision.
What might be the phylogenetic perspective on that, @cormacanderson and @PaulHeggarty?

**Comment by cormacanderson** [comment]
It's also my opinion that these cases should be treated as loanwords, whatever about the particulars of this or other Greek cases, where indeed the waters appear a bit muddier. I think the current policy with loanwords, whereby the loan source language is specified, should allow people to search for these cases, and, if desired, also to treat them differently than other loanwords.

**Comment by NikosLiosis** [comment]
I agree that the border between loans and inherited words in Modern Greek is blurry, at least in the context of diglossia, but sometimes the linguistic perpspective is not obvious either. For example the law for the deletion of the initial unstressed vowel has ceased to be functional a long time ago and did not affect every inherited word (due to analogy or other more specialized opposing forces). Therefore words like αγαπαω 'to love' and εγω 'Ι' have not lost their initial vowels (at least in SMG) but  nobody is ready to consider them register loanwords. There are similar problems regarding Tsakonian and (other) Modern Greek dialects as well.

**Comment by m-scarborough** [comment]
I hadn't thought about the case of εγώ; that's a very good point which confirms my general hesitance to go ahead marking all such words as loans.

**Comment by cormacanderson** [comment]
@m-scarborough meet @NikosLiosis, who is entering Tsakonian and a number of Greek dialects in CoBL. I think the two of you would have lots to talk about and hopefully we can get you together in Jena before too long :)
I believe the general policy here should be that, in principle, borrowings from an ancestral language should be treated as loanwords, e.g. Latin borrowings into Modern French should be treated in the same way as Latin borrowings into Modern English. There is enough information in the database, between clade assignations and the entry of the loan source language, to differentiate the two cases. If anyone thinks we should do this differently though, I'm open to suggestions.
More specifically however, the Greek instances seem to be a bit trickier. I think we might have to deal with them on a case by case basis, and provide justification in both the notes, as well as flagging the issue the eventual page introducing the language family. It is also something that should be discussed in any publication concerned with the Hellenic family in the database.
As @nitsnatsnok is currently going through the loanwords to try to get them to conform to most recent policy, I will now send an email to the three of you (Konstantin, Matthew, Nikos), so that you are all in touch and he can follow up about specific cases.
